api/routes/scripts/run.go

```diff
@@ -0,0 +1,22 @@
+package scripts
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+)
+
+// Run is the HTTP handler for the GET /scripts route.
+// It simply proxies the request to control-api.
+func Run(w http.ResponseWriter, r *http.Request) {
+	remote, err := url.Parse(fmt.Sprintf("%s/api", controlURL))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+	proxy := httputil.NewSingleHostReverseProxy(remote)
+	// We need to change the HOST header in order to pass control-api CORS.
```

Comment:
You can whitelist origins in the `django/river/settings/prod.py` as explained [here](https://github.com/adamchainz/django-cors-headers#cors_allowed_origins)